apps/sms/views/conversation/index.html

```diff
@@ -217,7 +217,7 @@ <h1 id="messages-header-text" aria-hidden="true"></h1>
                 <button id="messages-attach-button" aria-label="Add attachment" data-l10n-id="attach-button" type="button"></button>
               </div>
               <div class="composer-message-input-container">
-                <div id="messages-input" contentEditable="true" name="message"
+                <div id="messages-input" class="support-emojis" contentEditable="true" name="message"
```

Comment:
__nit:__ this view also contains subject field, message node (both text and subject) and report view (with subject in it).